DependencyInjection/Configuration.php

```diff
@@ -57,6 +57,18 @@ private function createAuthorizationServerNode(): NodeDefinition
                     ->cannotBeEmpty()
                     ->defaultValue('P1M')
                 ->end()
+                ->booleanNode('enable_client_credentials_grant')
+                    ->info('Whether to enable the client credentials grant')
+                    ->defaultTrue()
```

Comment:
@X-Coder264 Do we want the default to be true? Since we'll be bumping the major version it wouldn't be a bc break.

Reply:
@HypeMC It's true that we could make the default to be disabled as this PR will land in the next major version, but tbh I don't see any benefit in creating a breaking change in this case. From my perspective this PR aims to give the users the option to disable some grant if they really need to do it for whatever reason. Having all of them enabled by default seems ok too me. What's your take on this?

Reply:
@X-Coder264 The reason I'm asking is because it's very unlikely that someone will need more then one or two grant types, so it might be better to force the users to explicitly enable what they need instead of relying on them to know or remember to disable what they don't need.

Reply:
@HypeMC I don't like to force the users to do something that they really don't need to do. Having all grant types enabled seems (at least to me) like a sane default while having to disable a grant type seems like an edge case to me (and only then would the users have to check how to disable them, as opposed to the opposite scenario where each user would have to enable the grants they need as an additional step before they could begin to use the bundle). Laravel Passport also enables all grant types (except the implicit one) by default -> https://github.com/laravel/passport/blob/7.0/src/PassportServiceProvider.php#L95 That's just my 2c though.

Reply:
@X-Coder264 Works for me, just wanted to here you opinion on the matter.

Reply:
@HypeMC 👍